client/webserver/site/src/js/markets.js

```diff
@@ -262,6 +263,7 @@ export default class MarketsPage extends BasePage {
         Doc.hide(page.qtyBox)
       }
     }
+    this.previewOrder()
```

Comment:
Please just hide or show the `orderPreview` the same way we're doing with the other elements in this function.